packages/neo-one-smart-contract-lib/src/__tests__/ownership/TestOwnableContract.test.ts

```diff
@@ -0,0 +1,84 @@
+};
+
+describe('Ownable', () => {
+  test('deploy + transfer', async () => {
+    await withContracts<{ testOwnable: SmartContractAny }>(
+      [
+        {
+          filePath: path.resolve(__dirname, '..', '..', '__data__', 'contracts', 'ownership', 'TestOwnable.ts'),
+          name: 'TestOwnable',
+        },
+      ],
+      async ({ client, networkName, testOwnable: smartContract, masterAccountID }) => {
+        crypto.addPublicKey(
+          common.stringToPrivateKey(RECIPIENT.PRIVATE_KEY),
+          common.stringToECPoint(RECIPIENT.PUBLIC_KEY),
+        );
+
+        const deployResult = await smartContract.deploy(masterAccountID.address, { from: masterAccountID });
+        const deployReceipt = await deployResult.confirmed({ timeoutMS: 2500 });
+        if (deployReceipt.result.state !== 'HALT') {
+          throw new Error(deployReceipt.result.message);
+        }
+
+        expect(deployReceipt.result.gasConsumed.toString()).toMatchSnapshot('deploy consumed');
+        expect(deployReceipt.result.gasCost.toString()).toMatchSnapshot('deploy cost');
+        expect(deployReceipt.result.value).toBeTruthy();
+
+        const [initialOwner, recipient] = await Promise.all([
+          smartContract.owner(),
+          client.providers.memory.keystore.addUserAccount({
+            network: networkName,
+            name: 'recipient',
+            privateKey: RECIPIENT.PRIVATE_KEY,
+          }),
+        ]);
+
+        expect(initialOwner.toString()).toEqual(masterAccountID.address);
+
+        const transferReceipt = await smartContract.transferOwnership.confirmed(recipient.userAccount.id.address, {
+          from: masterAccountID,
+        });
+        const newOwner = await smartContract.owner();
+        if (transferReceipt.result.state !== 'HALT') {
+          throw new Error(transferReceipt.result.message);
+        }
+        expect(newOwner.toString()).toEqual(recipient.userAccount.id.address);
+
+        const bogusTransferReceipt = await smartContract.transferOwnership.confirmed(masterAccountID.address, {
+          from: masterAccountID,
+        });
+        expect(bogusTransferReceipt.result.state).toEqual('FAULT');
+        expect(bogusTransferReceipt.result.value).toBeFalsy();
+
+        const finalOwner = await smartContract.owner();
+        expect(finalOwner.toString()).toEqual(recipient.userAccount.id.address);
+
+        const renounceReceipt = await smartContract.renounceOwnership.confirmed({
+          from: recipient.userAccount.id,
+        });
+        expect(renounceReceipt.result.state).toEqual('HALT');
+        const noOwner = await smartContract.owner();
+        expect(noOwner).toEqual(undefined);
+
+        try {
+          const ownerOrThrow = await smartContract.ownerOrThrow.confirmed({
+            from: recipient.userAccount.id,
+          });
+          expect(ownerOrThrow.result.state).toBe('FAULT');
+        } catch {
+          expect(true).toBe(true);
```

Comment:
Do we expect an error to be thrown here? If so, use 

```
await expect(smartContract.ownerOrThrow.confirmed({
  from: recipient.userAccount.id,
})).rejects.toMatchSnapshot();
```

Reply:
.rejects.toBeDefined()